### ISSUE TITLE
Neogeo-menu is broken on multicart MVS

### ISSUE THREAD

**Issue by leonkiriliuk** [ArcadeTV/neogeo-menu#2]
When running neogeo-menu on a multicart (2 slot) MVS PCB, an illegal instruction is thrown every time a game is selected.
The address and most registers are the same.  Pressing "C" causes the error to go away and game plays.

If you put the 161-in-1 into slot 1, and a regular game into slot 2, after you press C, the selected 161-in-1 game loads
If you put the 161-in-1 into slot 2, and a regular game into slot 1, after you press C, the MVS game in slot 1 loads

<img width="425" alt="Screen Shot 2023-12-14 at 10 16 52 AM" src="https://github.com/ArcadeTV/neogeo-menu/assets/8943137/410f296f-b446-4bb5-b87b-7fa359d7cf28">
<img width="606" alt="Screen Shot 2023-12-14 at 10 18 04 AM" src="https://github.com/ArcadeTV/neogeo-menu/assets/8943137/856589c4-3c01-4e01-93f2-b5e409c75aec">

**Comment by mrehkopf** [comment]
Can you confirm that this only happens with this menu and not the stock menu on the same 161-in-1 Multicart? Did you also substitute the CP1 CPLD configuration from the VTXCart project or just the menu itself? I suspect that the CPLD is responding when it shouldn't. Might not be possible to fix because SLOTCS is not connected on the 161-in-1 carts.

**Comment by leonkiriliuk** [comment]
This doesn't happen with the stock 161-in-1 which has been in this arcade machine for months. VTXCart wouldn't even work if you don't reprogram all 3 FPGA's on the cart.  This cart works perfectly in 1 slot MVS. 

I did not try the VTXCart project menu which gets modified by the generator. I ordered another MVS donor to test this and it finally got here. I'm almost done building another cart with the stock menu to confirm that it's the menu code and not something else.

I don't think SLOTCS has anything to do with this problem. That pin is generated by NEO-F0 and goes low for the slot in use to signal to the game to function. If a cart ignores this signal, but is still in the active slot, it should function correctly (which this does) I think what happen is:

- Menu running fine
- game is selected (GSEL is set and communicated to all FPGAs)
- menu jumps to code to load game **- something goes wrong here**
- watchdog detects issue and resets the system (bad instruction triggered here as well)
- VTXCart via 0x2C0FEE knows what GSEL should be and has the game data addresses loaded rather than menu
- game plays after player presses continue on bad instruction menu

I wonder if these memory mapped registers have anything to do with this:
- REG_SYSTYPE or REG_SLOT
